checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsAnnotatedTypeFactory.java

```diff
@@ -77,6 +77,7 @@ public class CalledMethodsAnnotatedTypeFactory extends AccumulationAnnotatedType
    *
    * @param checker the checker
    */
+  @SuppressWarnings("nullness:method.invocation") // #979: inference failed
```

Comment:
This isn't an inference failure.  The error message is misleading for two reasons:
1. The Checker Framework infers wildcards for raw types and marks them as "uninferred wildcards" and prints `INFERENCE FAILED for:` before them.  I've fixe this in https://github.com/typetools/checker-framework/pull/6048 .
2. The method invocation warning is only checking the erased type of the receiver.  Annotations on receiver type arguments are ignored.  (https://github.com/typetools/checker-framework/issues/3203)

To fix the error message, the receiver of post init needs to be annotated so that it can be called in a constructor.  

All the warning suppressions need to be removed.